src/jobs/helm-argo-rollout.yml

```
@@ -0,0 +1,205 @@
+      condition: << parameters.s3-chart-repo >>
+      name: Add additional helm repo
+      steps:
+        - run: helmv3 repo add << parameters.s3-chart-repo >> s3://<< parameters.s3-chart-repo >>/
+
+  - run:
+      name: Generate ArgoCD Application values
+      command: |
+        cat \<< EOF > /tmp/argocd-app-values.yaml
+        argocd:
+          namespace: << parameters.argocd-namespace >>
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)